packages/rich-text/src/component/editable.js

```diff
@@ -46,6 +46,14 @@ class Editable extends Component {
 			element.reversed = nextProps.reversed;
 		}
 
+		if ( this.props.contentEditable !== nextProps.contentEditable ) {
```

Comment:
Does this need to be updated per changes in #17779 ? Should we be concerned that there was a use-case for toggling `contentEditable` that we didn't account for there? Considering we changed that behavior (https://github.com/WordPress/gutenberg/pull/17779#discussion_r345273235).

Reply:
Right, this PR really needs a rebase again. :)
I'm not concerned. I think we should mark the prop unstable as it would be for internal use only for now. There might be multiple features in future setting `contentEditable`, so I think it's actually really good to have it unstable.

* Multi-selection
* `disabled`
* `readonly`
* Template locking (would maybe use `disabled` or `readonly`)

Reply:
Rebased the PR